index.html

```diff
@@ -546,10 +547,14 @@ <h3>Integrity Validation</h3>
         <section>
           <h3>Evaluation</h3>
           <p>
-            Validation of a given Credential against its schema is to be performed according
-            to its associated [[JSON-SCHEMA]] specification. If validation
-            succeeds the credential is considered to be valid against its
-            <a>credential schema</a>.
+            Validation of a given credential against a schema is to be performed according
+            to its associated [[JSON-SCHEMA]] specification. Validation can result in three possible
```

Comment:
```suggestion
            to its associated [[JSON-SCHEMA]] specification. Validation MUST result in on of the following three possible
```

Reply:
fixes typo in @andresuribe87's suggestion

```suggestion
            to its associated [[JSON-SCHEMA]] specification. Validation MUST result in one of the following three possible
```